------------------------------------------------------------
revno: 741
fixes bug: https://launchpad.net/bugs/1080717
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Mon 2012-11-19 09:25:09 -0500
message:
  add 'sudo' entry for default user in default config/cloud.cfg
=== modified file 'ChangeLog'
--- a/config/cloud.cfg
+++ b/config/cloud.cfg
@@ -82,6 +82,7 @@ system_info:
      lock_passwd: True
      gecos: Ubuntu
      groups: [adm, audio, cdrom, dialout, floppy, video, plugdev, dip, netdev]
+     sudo: ["ALL=(ALL) NOPASSWD:ALL"]
      shell: /bin/bash
    # Other config here will be given to the distro class and/or path classes
    paths:
